push-img.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# Push a specified image to
+# the Docker Hub.
+set -eu
+
+if [ "$#" -ne 1 ]; then
+    printf "Usage: ./push-img <dirname>
+where <dirname> is the name of a
+directory in this repository."
+    exit
+fi
+
+BASENAME=$1
+LOCALIMAGENAME=oqs-"${BASENAME}"
+
+cd "${BASENAME}"
+
+docker build -t "${LOCALIMAGENAME}"
+
+printf "Pushing image to openqsafe."
+docker tag "${LOCALIMAGENAME}" openquantumsafe/ci-"${BASENAME}"
+docker push openquantumsafe/ci-"${BASENAME}"
```

Comment:
To be clear on the repository and tag naming, we would then have a repository `openquantumsafe/ci-centos-8` with a `oqs-centos-8` tag in it?

Reply:
No, the names I used were just placeholders.  I'll think about it some more and revise it.